ansible/elastic.yml

```diff
@@ -0,0 +1,13 @@
+---
+
+- hosts: all
+  become: yes
+  
+  pre_tasks:
+    
+    - name: Run init tasks
+      import_tasks: tasks/init.yml
+    
+  roles:
+    - { role: rsyslog_client }
+    - { role: elastic, elastic_version: 6.x }
```

Comment:
elastic_version should not be passed there, it comes from the config. Instead, specify a defaults in the role itself.